### PR TITLE
Nodejs week1 HW

### PR DESCRIPTION
**Pull request by Deepti-Dk**

### CODE REVIEW

**Review by kfazinic — CHANGES_REQUESTED** [review]
Very nice job! 👏 

Have you tried running tests? One of them fails, try to fix it! 🙂

File: nodejs/week1/MealSharing/src/backend/app.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+});
+app.get('/cheap-meals', (request, response) => {
+  response.send(cheapMeal);
+});
+app.get('/large-meals', (request, response) => {
+  response.send(largeMeal);
+});
+app.get('/meal', (request, response) => {
+  response.send(meal);
+});
```
[the remark this answers is not in the document]
Comment (Deepti-Dk) [reply nodejs/week1/MealSharing/src/backend/app.js:22]: Hey Kristine, I did npm install -D nodemon and npm install nodemon -g to sort meal.js and reservation.js to display different data on refresh but its still displaying same ones. Don't know what more I could do! Please advise:-)

[the remark this answers is not in the document]
Comment (Deepti-Dk) [reply nodejs/week1/MealSharing/src/backend/app.js:22]: --Sorted--

**Review by kfazinic — APPROVED** [review]
👏

**Pull request merged**